code/go/0chain.net/blobbercore/handler/grpchandler.go

```diff
@@ -0,0 +1,90 @@
+package handler
+
```

Comment:
Is there any to unit test this. Maybe use [httptest](https://golang.org/pkg/net/http/httptest/) to send mock requests and make sure they reach your new `GetAllocation` endpoint?

Reply:
I was thinking use httptest to create mock http requests, create a mock set of endpoints that record when they are called, then you can map requests to endpoint calls.,

Reply:
It looks like PR https://github.com/0chain/blobber/pull/61 can work as a unit for this PR.

Reply:
I will add unit tests for this and all other endpoints

Reply:
Since the changes in #61 can be considered as unit tests, we can resolve this